compiler/natives/src/crypto/ed25519/internal/edwards25519/scalar_test.go

```diff
@@ -0,0 +1,22 @@
+//go:build js
+// +build js
+
+package edwards25519
+
+import "testing"
+
+func TestScalarMultDistributesOverAdd(t *testing.T) {
+	t.Skip("slow") // Times out, takes ~13 minutes
```

Comment:
Not related to this PR per se, but thinking out loud. One reason I heard for people using GopherJS was access to the crypto library, and our crypto in incredibly slow. I did a little profiling and it seems to all boil down to `math/big` being really slow with our implementation of slices. I couldn't think of an easy way to improve the situation, but these tests shall serve as additional motivation to try and optimize it further.